include/odp/api/spec/ipsec.h

```diff
@@ -831,16 +831,17 @@ odp_ipsec_sa_t odp_ipsec_sa_create(const odp_ipsec_sa_param_t *param);
  * before calling disable. Packets in progress during the call may still match
  * the SA and be processed successfully.
  *
- * When in synchronous operation mode, the call will return when it's possible
- * to destroy the SA. In asynchronous mode, the same is indicated by an
- * ODP_EVENT_IPSEC_STATUS event sent to the queue specified for the SA. The
- * status event is guaranteed to be the last event for the SA, i.e. all
- * in-progress operations have completed and resulting events (including status
- * events) have been enqueued before it.
+ * When SA is disabled synchronously, the call will return when it's possible
+ * to destroy the SA indicated by return value 0. If the return value is 1,same
+ * is indicated by an ODP_EVENT_IPSEC_STATUS event sent to the queue specified
+ * for the SA. The status event is guaranteed to be the last event for the SA,
+ * i.e. all in-progress operations have completed and resulting events
+ * (including status events) have been enqueued before it.
```

Comment:
Suggested simpler wording:

A return code of 0 indicates the disable request has completed synchronously and is the SA is now disabled. A return code of 1 indicates that the disable request has been accepted and completion will be indicated by receipt of an ODP_EVENT_IPSEC_STATUS sent to the queue specified for the SA. This event is guaranteed to be the last event for the SA, i.e., all in-progress operations have completed and resulting events (including status events) have been enqueued before it.